index.html

```diff
@@ -670,6 +670,25 @@ <h3>
 ]
 </code></pre>
 <p>You can open the RabbitMQ's management console and find these messages in a queue.</p>
+
+<h2>Logging</h2>
+
+
+<p>REMReM Publish application logging is implemented using the logback-classic, and we required to configure the logback.xml file. <br/>
```

Comment:
logback-classic. It requires user to configure the logback.xml file.
When using without logback.xml log messages will look as below:
To configure the logback.xml file use  <strong>-Dlogging.config=path/logback.xml</strong>


To get sample logback.xml to use Click Here